apps/balancer-tools/src/app/apr/(utils)/tokenYield.ts

```diff
@@ -0,0 +1,61 @@
+import { NetworkChainId } from "@bleu-balancer-tools/utils";
+import { createPublicClient, http } from "viem";
+import { mainnet } from "viem/chains";
+
+import { blocks } from "#/lib/gql/server";
+
+const rateProviderAbi = [
+  {
+    inputs: [],
+    name: "getRate",
+    outputs: [{ internalType: "uint256", name: "", type: "uint256" }],
+    stateMutability: "view",
+    type: "function",
+  },
+];
+
+export const publicClient = createPublicClient({
+  chain: mainnet,
+  transport: http(),
+});
+
+const SECONDS_IN_DAY = 86400;
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
remove this on https://linear.app/bleu-llc/issue/BAL-791/interface-integrate-yield-apr-function-with-the-app